docs/code-reference.rst

```diff
@@ -0,0 +1,20 @@
+.. _code_reference:
+
+==============
+Code Reference
+==============
+
+.. automodapi:: glue_solar.core.core
```

Comment:
```suggestion
.. automodapi:: glue_solar.core
```

Should be enough since the init file imports everything from core.

Reply:
Sure, and I think then I can remove the other similar ones starting with the pattern `glue_solar.core` as well...